src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20240416_add_sam_rac_tables.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="kfeldman" id="add_sam_rac_tables">
+        <createTable tableName="SAM_RESOURCE_ACCESS_CONSTRAINT">
+            <column name="id" type="BIGINT" autoIncrement="true">
```

Comment:
change this (and other ids) to type `uuid` and let postgres create them with gen_random_uuid